### PR TITLE
Navbar added to the header of webpage

### PR DESCRIPTION
**Pull request by agustin-pomies**

### CODE REVIEW

**Comment by santiago-bertinat** [comment]
Check your commits when you do a rebase. You have duplicated commits in this branch

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (santiago-bertinat) [inline app/assets/javascripts/static_pages.coffee]: remove if it is not used

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the static_pages controller here.
```
Comment (santiago-bertinat) [inline app/assets/stylesheets/static_pages.scss]: remove all unused files

File: app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -1,26 +1,28 @@
-<h2>Log in</h2>
+<div class='central'>
```
Comment (santiago-bertinat) [inline app/views/devise/sessions/new.html.erb]: maybe `centered-container` would be a better name, what do you think?

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<header class='navbar-header'>
+  <div class='container'>
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <ul class='nav navbar-nav'>
+      <li id='home'>
```
Comment (santiago-bertinat) [inline app/views/layouts/_navbar.html.erb]: do you need to put ids to all li? don't they have the same style?

File: app/views/tasks/index.html.erb [hunk 5]
```diff
@@ -1,57 +1,60 @@
...
-          <% end %>
+  <h2>Completed tasks</h2>
+  <ul class='tasks-lists complete'>
+    <% @tasks.each do |task| %>
+      <% if task.completed %>
+        <li>
+          <div class='wrapper'>
+            <%= task.description %>
+            <%= check_box_tag 'tasks_ids[]', task.id, task.completed, class: 'check' %>
+            <% if task.assignments.exists?(user_id: current_user.id, ownership: true) %>
```
Comment (santiago-bertinat) [inline app/views/tasks/index.html.erb]: if you reuse this query you can use a helper method to encapsulate it

File: test/controllers/static_pages_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```
Comment (santiago-bertinat) [inline test/controllers/static_pages_controller_test.rb]: remove all autogenerated files that are not used

File: app/views/layouts/_navbar.html.erb (hunk 4 above)
Comment (agustin-pomies) [reply app/views/layouts/_navbar.html.erb]: The <li> tags have the same style at first sight, but each one of them has a unique background color when you hover the coursor on them

**Pull request merged**